packages/LSOnline/modules/player/playerEvents.js

```diff
@@ -65,10 +55,46 @@ mp.events.add({
   },
 
   playerChat: (player, text) => {
+    if (!player.isLogged) {
+      return player.kick();
+    }
+
     if (player.brutallyWounded) {
       return player.call('actionDone', ['Nie możesz tego teraz użyć!', 'Twoja postać jest nieprzytomna!']);
     }
 
     mp.players.broadcastInRange(player.position, 25, player.dimension, `${player.name} mówi: ${text}`);
   }
 });
+
+const createQuitLabel = (player, exitType) => {
+  if (!player.quitLabel) {
+    player.quitLabel = mp.labels.new(`~HUD_COLOUR_GREYLIGHT~ (( ${player.name} - ${exitType} ))`, new mp.Vector3(player.position.x, player.position.y, player.position.z),
+      {
+        los: true,
+        font: 0,
+        drawDistance: 10,
+        dimension: player.dimension
+      });
+  }
+
+  // This need to be rewrited when we gonna create labels manager.
+  setTimeout(() => {
+    if (player.quitLabel) {
+      player.quitLabel.destroy();
+    }
+  }, 50000);
+};
+
+const prepareBeforeQuit = (player, exitType = false) => {
+  clearBrutallyWoundedTimers(player);
+
+  if (player.vehicle) {
+    const vehicle = { id: player.vehicle.informations.id, seat: player.seat };
+    player.character.saveBeforeQuit(player, vehicle, exitType);
+  } else {
+    player.character.saveBeforeQuit(player, undefined, exitType);
+  }
+};
+
+exports.prepareBeforeQuit = prepareBeforeQuit;
```

Comment:
Unnecessary exports. Should be also module.exports = {}

Reply:
https://medium.freecodecamp.org/node-js-module-exports-vs-exports-ec7e254d63ac

I dont see any unnecessary exports.